src/amo/components/PermissionsCard/styles.scss

```diff
@@ -0,0 +1,29 @@
+.PermissionCard-learn-more {
+  margin: 12px auto 0;
+  width: 100%;
+
+  .Icon {
+    margin-left: 10px;
+  }
+}
+
+.PermissionsCard-subhead {
+  color: #737373;
+}
+
+.PermissionsCard-list {
+  list-style: none;
+  margin: 10px 0 0;
```

Comment:
The top margin I see in Sketch is `18px` but this can probably be set to `0` since the `li` will already have a top margin of `18px` (which I mentioned up above).